src/TogglePassword/templates/form_theme.html.twig

```diff
@@ -0,0 +1,8 @@
+{%- block password_widget -%}
+    {%- set type = type|default('password') -%}
+    {%- if toggle is defined -%}
+        <div class="toggle-password-container">{{ block('form_widget') }}</div>
```

Comment:
Maybe make this class configurable as well?

Reply:
You're right, it was the last element unavailable for customization i think 😄